README.md

```diff
@@ -1 +1,139 @@
-# math-magic
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>Math Magicians App</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Math-Magician]
+
+> Describe your project in 1 or 2 sentences.
+
+**[Math Magician]** 
+-This project is math magician for calculating figures
+
+## 🛠 Built With
+-React
+
+<!-- Features -->
+
+### Key Features
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
```

Comment:
### Optional
- [ ] It would be nice if we remove the “Key_feature_1” it is just a placeholder and it’s not making our README file professional